homework02/go.mod

```diff
@@ -0,0 +1,3 @@
+module Golang/epam/goCourse
```

Comment:
Should be `module Golang/epam/goCourse/homework02`

Reply:
updated module Golang/epam/goCourse/homework02

Reply:
What should be my next step in this PR - merge PR or close PR?